api/src/main/java/org/terrakube/api/plugin/scheduler/job/tcl/executor/ephemeral/EphemeralExecutorService.java

```diff
@@ -80,6 +78,31 @@ public boolean sendToEphemeralExecutor(Job job, ExecutorContext executorContext)
                 executorContext.getEnvironmentVariables().containsKey(SERVICE_ACCOUNT) ? executorContext.getEnvironmentVariables().get(SERVICE_ACCOUNT) : null);
         String serviceAccount = serviceAccountInfo.isPresent() ? serviceAccountInfo.get() : null;
 
+        // Volume and VolumeMount for ConfigMap if specified
+        List<Volume> volumes = new ArrayList<>();
+        List<VolumeMount> volumeMounts = new ArrayList<>();
+
+        Optional<String> configMapNameOpt = Optional.ofNullable(executorContext.getEnvironmentVariables().get("CONFIG_MAP_NAME"));
+        Optional<String> configMapMountPathOpt = Optional.ofNullable(executorContext.getEnvironmentVariables().get("CONFIG_MAP_MOUNT_PATH"));
+        if (configMapNameOpt.isPresent()) {
+            String configMapName = configMapNameOpt.get();
+            String mountPath = configMapMountPathOpt.orElse("/home/");  // Default mount path if not specified
```

Comment:
I would move the default mount path to `/tmp` or `/mnt`. From my POV `/home` is not safe to be used as a mount point.

Reply:
I set it as default because it is used by AWS for instance to fetch its .aws/config folder but you're right as a safe point i'll move to tmp !